migration_spinner/command_line.py

```diff
@@ -0,0 +1,27 @@
+import os
+import site
+
+import spinner
+import argparse
+
+
+def check_airflow_package(value):
+    if not os.path.exists(value):
+        raise argparse.ArgumentTypeError("Airflow was not found in path: %s" % value)
+    return value
+
+
+def main():
+    default_airflow_package_dir = os.path.join(site.getsitepackages()[0], 'airflow')
```

Comment:
Rather than having to specify it we can use importlib to ask where it lives:

```
In [9]: importlib.util.find_spec('airflow').origin
Out[9]: '/Users/ash/code/python/incubator-airflow/airflow/__init__.py'
```

Reply:
fixed.